sources/iTermRootTerminalView.m

```diff
@@ -16,7 +16,7 @@
 #import "iTermToolbeltView.h"
 #import "PTYTabView.h"
 
-const CGFloat kHorizontalTabBarHeight = 22;
+const CGFloat kHorizontalTabBarHeight = 24;
```

Comment:
This is a somewhat disruptive change because it will affect peoples' window arrangements, by making windows taller or causing them to lose a row in some cases. It is nice to get closer to the system tab bar's look, but it's going to cause some pain. I just want to think about this some more.